static/src/js/components/Tour/TourSteps.jsx

```diff
@@ -94,507 +93,587 @@ const TourSteps = ({
-                className="search-tour__checkbox"
-                checked={isChecked}
-                onChange={handleCheckboxChange}
-              />
-              <label htmlFor="dontShowAgain" className="search-tour__checkbox__label">
-                Don&apos;t show again
-              </label>
-            </div>
-            <div className="search-tour__buttons">
-              <Button
-                type="button"
-                bootstrapVariant="secondary"
-                bootstrapSize="sm"
-                onClick={() => setStepIndex(stepIndex - 1)}
-              >
-                Previous
-              </Button>
-              <Button
-                className="button-tour-finish"
-                type="button"
-                bootstrapVariant="primary"
-                bootstrapSize="sm"
-                onClick={
-                  () => {
-                    setStepIndex(stepIndex + 1)
+      ),
+      placement: 'left',
+      hideFooter: true,
+      styles: commonStyles
+    },
+    {
+      target: '.secondary-toolbar__user-dropdown-toggle',
+      content: (
+        <div className="search-tour__content-wrapper">
+          <StepCounter currentStep={stepIndex} totalSteps={TOTAL_STEPS} />
+          <p className="search-tour__content">
+            Use this menu to set preferences, view saved projects, manage subscriptions,
+            view download status and history, or log out.
+          </p>
+          <TourButtons
+            stepIndex={stepIndex}
+            setStepIndex={setStepIndex}
+            isChecked={isChecked}
+            handleCheckboxChange={handleCheckboxChange}
+          />
+        </div>
+      ),
+      placement: 'left',
+      hideFooter: true,
+      styles: commonStyles
+    }
+  ]
+
+  const loggedOutStep = [
+    {
+      target: '.search',
+      content: (
+        <div className="search-tour__content-wrapper">
+          <StepCounter currentStep={stepIndex} totalSteps={TOTAL_STEPS} />
+          <p className="search-tour__content">
+            Log in with Earthdata Login to set preferences, save projects, create and
+            manage subscriptions, and view download status and history.
+          </p>
+          <TourButtons
+            stepIndex={stepIndex}
+            setStepIndex={setStepIndex}
+            isChecked={isChecked}
+            handleCheckboxChange={handleCheckboxChange}
+          />
+        </div>
+      ),
+      placement: 'center',
+      hideFooter: true,
+      styles: commonStyles,
+      disableBeacon: true
+    }
+  ]
+
+  const postSteps = [
+    {
+      target: '.secondary-toolbar__start-tour-button',
+      content: (
+        <div className="search-tour__content-wrapper">
+          <StepCounter currentStep={stepIndex} totalSteps={TOTAL_STEPS} />
+          <p className="search-tour__content">
+            You can replay this tour anytime by clicking
+            {' '}
+            <strong>Show Tour</strong>
```

Comment:
We missed this in the last PR, but use the ? icon here like we do in the first step